src/ext.js

```diff
@@ -536,6 +530,50 @@ export default function ext({ translator }) {
                 },
               },
             },
+            icon: {
+              type: 'items',
+              grouped: true,
```

Comment:
I'm not sure this does anything. If it doesn't we can remove all uses of it in ext